src/parcsr_ls/par_cr.c

```diff
@@ -2076,7 +2075,9 @@ hypre_BoomerAMGIndepPMIS( hypre_ParCSRMatrix    *S,
          }
       }
 
+#if 0 /* debugging */
```

Comment:
Is there a reason you are guarding this with `#if 0` instead of just eliminating (or commenting out) as you've done in other places?

Reply:
Yes.  Ulrike commented that this looked like debugging stuff and should be marked as such.  So, I used the same `#if 0` line that was already there in other parts of the file.  It would be better if there was a `DEBUG` flag, but I took the path of least effort and followed what was already there. :)